sensor_msgs/Image detected_image
---
custom_server/custom answer


